src/content/developers/docs/networks/index.md

```diff
@@ -48,25 +50,25 @@ A proof-of-work testnet. The best like-for-like representation of Ethereum Mainn
 - [Sepolia faucet](https://faucet.sepolia.dev/)
 - [FaucETH](https://fauceth.komputing.org)
 
-#### Görli {#goerli}
+#### Goerli {#goerli}
```

Comment:
I added a commit to update these to all use "Goerli" instead of "Görli"... Not a huge personal preference, but this started getting rolled out over on the Launchpad and figure it's best to be consistent.